File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/42_compito_python_20230116/Articolo.py
class Articolo:
    def __init__(self, codice, tipo, marca, taglia, modello, costo, sconto):
        self._codice = codice
        self._tipo = tipo
        self._marca = marca
        self._taglia = taglia
        self._modello = modello
        self._costo = costo
        self._sconto = sconto

    def get_costo(self):
        return self._costo

    def __str__(self):
        return self._tipo + "\t" + str(self._codice) + "\t" + self._marca + "\t" + self._modello + "\t" + self._taglia

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/42_compito_python_20230116/Articolo.java
public class Articolo {
    protected int codice;
    protected String tipo;
    protected String marca;
    protected String taglia;
    protected String modello;
    protected float costo;
    protected int sconto_in_percentuale;

    public Articolo(int codice, String tipo, String marca, String taglia, String modello, float costo,
            int sconto_in_percentuale) {
        this.codice = codice;
        this.tipo = tipo;
        this.marca = marca;
        this.taglia = taglia;
        this.modello = modello;
        this.costo = costo;
        this.sconto_in_percentuale = sconto_in_percentuale;

    }

    public String toString() {
        return tipo + "\t" + codice + "\t" + marca + "\t" + modello + "\t" + taglia;
    }
}